server/controllers/admin.py

```diff
@@ -1437,9 +1437,22 @@ def client(client_id):
 # Student View #
 ################
 
-@admin.route("/course/<int:cid>/<string:email>")
+@admin.route("/course/<int:cid>/<string:email>", methods=['GET', 'POST'])
 @is_staff(course_arg='cid')
 def student_view(cid, email):
+
+    form = forms.EnrollmentForm()
+    if form.validate_on_submit():
+        if form.email.data != email:
+            flash("You cannot change the user's email.", 'error')
+        else:
+            email, role = form.email.data, form.role.data
+            Enrollment.enroll_from_form(cid, form)
+            flash("Edited User Successfully", 'success')
```

Comment:
Nit, prefer single quotes unless a single quote is used in the string. (I know this isn't consistent everywhere).